lib/samples/show_realistic_light_and_shadows/show_realistic_light_and_shadows.dart

```diff
@@ -0,0 +1,198 @@
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    // dropdown for lighting choices
+                    DropdownButton(
+                      value: _selectedLighting,
+                      items: _lightingChoices.keys
+                          .map(
+                            (label) => DropdownMenuItem(
+                              value: label,
+                              child: Text(
+                                label,
+                                style: TextStyle(
+                                  color: Theme.of(context).primaryColor,
+                                ),
+                              ),
+                            ),
+                          )
+                          .toList(),
+                      onChanged: (value) {
+                        if (value != null) {
+                          setState(() {
+                            _selectedLighting = value;
+                            // Update the sun lighting based on the selected choice.
+                            _sceneViewController.sunLighting =
+                                _lightingChoices[value]!;
+                          });
+                        }
+                      },
+                    ),
+                  ],
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    // Slider to change the sun time
+                    Expanded(
+                      child: Padding(
+                        padding: const EdgeInsets.fromLTRB(20, 0, 20, 0),
+                        child: Slider(
+                          max: 24,
+                          divisions: 24 * 4, // 15-minute increments
+                          value: _timeValue,
+                          label: '${_timeValue.toStringAsFixed(2)} h',
+                          onChanged: (value) {
+                            setState(() {
+                              _timeValue = value;
+                              // Update the sun time based on the slider value.
+                              setSunTimeFromValue(_timeValue);
+                            });
+                          },
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    // Slider to change the sun time from 0 to 24 hours
+                    Padding(
+                      padding: const EdgeInsets.only(bottom: 10),
+                      child: Text(
+                        // Format the sun time as MM dd, yyyy, hh:mm a
+                        // ignore: unnecessary_string_interpolations
+                        '${_dateFormat.format(_sunTime)}',
```

Comment:
```suggestion
                        // Format the sun time as MM dd, yyyy, hh:mm a
                        ${_dateFormat.format(_sunTime)},
```
I think that `format` method returns a `String` - can this be simplified?